.github/workflows/docker_publish.yml

```diff
@@ -6,6 +6,7 @@ name: Docker Publish
 # documentation.
 
 on:
+  pull_request:
```

Comment:
💭 Do we want to verify that Docker builds properly on a PR? (no push occurs)

Reply:
I don't think it's necessary, as all it really does is run npm install, but I don't feel too strongly about it. If we do want to keep it, we'll need to update something with the build and push step as it looks like the latest run pushed to the latest tag.

Reply:
Will remove this!